src/main/java/uk/gov/hmcts/document/DocumentClientApi.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.document;
+
+import feign.Headers;
+import feign.Param;
+import feign.RequestLine;
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.document.domain.UploadResponse;
+
+@FeignClient(name = "document.management.gateway.api", url = "${document.management.gateway.api.url}")
+public interface DocumentClientApi {
+
+    @RequestLine("POST /documents")
```

Comment:
we should be using the spring annotations not the feign ones

Reply:
Feign Test doesn't work with spring annotations